### PR TITLE
Moon15: Steel Market Monopolists and Titanium Market Monopolists

### PR DESCRIPTION
**Pull request by kberg**

### CODE REVIEW

File: src/cards/moon/AIControlledMineNetwork.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import {CardType} from '../CardType';
+import {IProjectCard} from '../IProjectCard';
+import {Tags} from '../Tags';
+import {CardMetadata} from '../CardMetadata';
+import {MoonExpansion} from '../../moon/MoonExpansion';
+import {CardRenderer} from '../render/CardRenderer';
+import {CardRequirements} from '../CardRequirements';
+
+export class AIControlledMineNetwork implements IProjectCard {
+  public cost = 6;
```
Comment (bafolts) [inline src/cards/moon/AIControlledMineNetwork.ts:12]: This should use the static card property pattern.

File: src/cards/moon/PrideoftheEarthArkship.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          }),
+          b.minus().titanium(2);
+        }),
+        victoryPoints: CardRenderDynamicVictoryPoints.science(1, 1),
+      },
+    }, {
+      reserveUnits: Units.of({titanium: 2}),
+    });
+  };
+  public readonly resource = ResourceType.SCIENCE;
```
Comment (bafolts) [inline src/cards/moon/PrideoftheEarthArkship.ts:36]: Is this separate `resource` property used?

**Review by bafolts — APPROVED** [review]

File: src/cards/moon/AIControlledMineNetwork.ts (hunk 1 above)
Comment (kberg) [reply src/cards/moon/AIControlledMineNetwork.ts:12]: Whoops! Fixed.

File: src/cards/moon/PrideoftheEarthArkship.ts (hunk 2 above)
Comment (kberg) [reply src/cards/moon/PrideoftheEarthArkship.ts:36]: Whoops. Removing.

**Pull request merged**